packages/tools/src/envinfo.ts

```diff
@@ -35,7 +35,7 @@ async function getEnvironmentInfo(
       Binaries: ['Node', 'Yarn', 'npm', 'Watchman'],
       IDEs: ['Xcode', 'Android Studio', 'Visual Studio'],
       Managers: ['CocoaPods'],
-      Languages: ['Java', 'Python'],
+      Languages: ['Java'],
```

Comment:
Please ignore this change. Did it to make it easier to rebase.

Reply:
When you move `envinfo` where it was, just nevert this change too :)

Reply:
Python is no longer necessary on Windows, so no need to declare it here, I'd keep it removed :)

Reply:
Yes, but this code is not based on latest master and at the time of cutting the original branch, Python was still in the source. I removed it hoping that Github UI would let me resolve conflicts in the UI, otherwise it says that they're too complex.

Either way, let's just revert all changes done to this file in this PR and Github will do the rest.